From 9d0a3eb98982c38bfe1e1872498538b103ad06cd Mon Sep 17 00:00:00 2001
From: Marien Zwart <marienz@google.com>
Date: Sun, 7 Apr 2024 17:46:36 +1000
Subject: [PATCH] Do not call locate-user-emacs-file when compiling

The default value of phpactor-install-directory is relative to the
user's user-emacs-directory. Using eval-when-compile here hardcodes the
wrong directory if a separate build user is used.
---
 phpactor.el | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/phpactor.el b/phpactor.el
index 7e7c036..96de425 100644
--- a/phpactor.el
+++ b/phpactor.el
@@ -68,8 +68,7 @@
 
 ;;;###autoload
 (defcustom phpactor-install-directory
-  (eval-when-compile
-    (expand-file-name (locate-user-emacs-file "phpactor/")))
+  (expand-file-name (locate-user-emacs-file "phpactor/"))
   "Directory for setup Phactor.  (default `~/.emacs.d/phpactor/')."
   :type 'directory)
 
-- 
2.42.0

